src/sign/participant.rs

```diff
@@ -156,39 +158,69 @@ impl ProtocolParticipant for SignParticipant {
     where
         Self: Sized,
     {
-        todo!()
+        let config = ParticipantConfig::new(id, &other_participant_ids)?;
```

Comment:
note: There's something very inefficient here where the `ProtocolParticipant` constructors have to take `id` and `other_participant_id`s separately, but they should really just take our well-defined and validated `ParticipantConfig` type. I'll make a separate issue to clean this up. Also the `ParticipantConfig::new` constructor takes the `other_id`s by reference and then copies them all into a `vec` which is not really our standard approach (constructors should take stuff by value if they're going to use it).